string name
uint64 read_count 
uint64 write_count 
uint64 read_bytes 
uint64 write_bytes 
uint64 read_merged_count 
uint64 write_merged_count 
uint64 read_time
uint64 write_time
uint64 busy_time 
rcp/DiskPartition[] partitions